google-cloud-monitoring/lib/google/cloud/monitoring.rb

```diff
@@ -170,7 +167,7 @@ def self.new(*args, version: :v3, **kwargs)
       module Group
         ##
         # The Group API lets you inspect and manage your
-        # [groups](https://cloud.google.comgoogle.monitoring.v3.Group).
+        # [groups](https://cloud.google.com#google.monitoring.v3.Group).
```

Comment:
Is this URL correct?

Reply:
No it's not, but it's better than what was there before. I need to track that one down internally.